src/pages/play.astro

```diff
@@ -222,19 +258,28 @@ const t = useTranslations(lang)
     const footer = document.createElement('div')
     footer.classList.add('log-footer')
 
-    const disableButton = document.createElement('button')
-    disableButton.textContent = 'Disable'
-    disableButton.classList.add('btn', 'tools-secondary-btn')
-    disableButton.addEventListener('click', () => {
+    const switchButton = document.createElement('div')
+    switchButton.innerHTML = `
+      <label class="wallet-switch">
```

Comment:
Nit: let's create `const switchButton = document.createElement('label')` and skip the extra `div`?